Read node status when querying a baker (#568)
Concordium/concordium-scan: backend-rust/CHANGELOG.md

### Added

> - Add `Node::Baker` which returns node information on a validator

Reviewer: ```suggestion
- Add `Baker::node_status` to retrieve the validator's node information.
```
